.evergreen/debian_package_build.sh

```diff
@@ -41,10 +41,10 @@ cd ..
 
 git clone https://salsa.debian.org/installer-team/debootstrap.git debootstrap.git
 export DEBOOTSTRAP_DIR=`pwd`/debootstrap.git
-# perl-openssl-defaults is explicitly listed to work around https://bugs.debian.org/907015
```

Comment:
Ah nice, since that bug was closed, I suppose we no longer need `perl-openssl-defaults`.

Reply:
That's correct.